test/test_diagnostics.jl

```diff
@@ -13,23 +13,23 @@ function run_horizontal_average_tests(arch, FT)
 
     T̅ = Average(model.tracers.T, dims=(1, 2), with_halos=false)
     computed_T_profile = T̅(model)
-    @test size(computed_T_profile) == (1, 1, Nz)
-    @test computed_T_profile ≈ znodes(Cell, grid, reshape=true)
+    @test_skip size(computed_T_profile) == (1, 1, Nz)
+    @test_skip computed_T_profile ≈ znodes(Cell, grid, reshape=true)
```

Comment:
Do these tests fail right now or are you skipping them in preparation for `Average` to be overhauled?

Reply:
I am skipping them because the tests started timing out.

Reply:
Hmmm, these were small fast tests and most of the setup still occurs but we can re-enable once testing doesn't time out.

Reply:
Ah, I didn't look at the run time. I just noticed that adding the (largely identical) tests for `AveragedField` made the integration group time out. So I skipped these, and the tests stopped timing out. But maybe I just got lucky?